shared/studio/tabs/dataview/dataInspector.tsx

```diff
@@ -951,3 +1002,80 @@ const ExpandedDataInspector = observer(function ExpandedDataInspector({
     </div>
   );
 });
+
+interface MobileDataInspectorProps {
+  expandedState: ExpandedInspector;
+}
+
+const MobileDataInspector = observer(function MobileDataInspector({
+  expandedState,
+}: MobileDataInspectorProps) {
+  const item = expandedState.getItems()?.[0] as ObjectLikeItem | undefined;
+
+  const state = useDataInspectorState().state;
+  const fields = state.fields || [];
+
+  const {navigate, currentPath} = useDBRouter();
+  const basePath = currentPath.join("/");
+
+  const closeExtendedView = () => {
+    state.setExpandedRowMobile(null);
+  };
+
+  return (
+    <div className={styles.mobileInspectorWindow}>
+      <div className={styles.header}>
+        <button onClick={closeExtendedView} className={styles.headerBtn}>
+          <BackIcon />
+        </button>
+
+        <p className={styles.title}>
+          {item ? item.data.__tname__ : `loading...`}
+        </p>
+        <button onClick={closeExtendedView} className={styles.headerBtn}>
+          <CrossIcon />
+        </button>
+      </div>
+      {item &&
+        fields.map((field, index) => {
+          const isMulti = field.multi;
+          const data = item.data;
+          const value = isMulti ? data[field.name].length : data[field.name];
```

Comment:
`isMulti` is the wrong thing to check here to differentiate between props and links (links can be single and props can be multi). Also in the query links are fetched with `limit 10`, so you need to display the `__count_*` field it fetches instead.

```suggestion
          const value = field.type === ObjectFieldType.link ? data[`__count_${field.name}`] : data[field.name];
```